chain/network/src/peer_manager/tests/connection_pool.rs

```diff
@@ -310,3 +320,93 @@ async fn invalid_edge() {
         }
     }
 }
+
+async fn test_signed_owned_ip_address(expected_closing_reason: ClosingReason, wrong_ip_address: &Option<std::net::IpAddr>, wrong_node_key: &Option<near_crypto::SecretKey>) {
+    init_test_logger();
+    let mut rng = make_rng(921853233);
+    let rng = &mut rng;
+    let mut clock = time::FakeClock::default();
+    let chain = Arc::new(data::Chain::make(&mut clock, rng, 10));
```

Comment:
This test is quite long and it would benefit from some visual segmentation for readability. For example, the other tests in this file put an empty line after these first four lines to visually separate the setup of these basic variables from the rest of the test-specific code.

Reply:
Added some white spaces, hopefully that helps.

Reply:
Not sure which test you changed but `test_backward_compatible_handshake_without_signed_ip_address` still looks to have this issue.

Reply:
Ah lol, the test `test_backward_compatible_handshake_without_signed_ip_address` is in a different file, this comment is pointing towards connection_pool.rs whereas that test is in the file communication.rs 

I previously updated the `test_signed_ip_address` which is being referred to by this code review comment. 

Updated the test as suggested, however, do note that this backward compatibility test is just temporary and will be removed once all nodes have fully migrated to signing ip address.